src/jllvm/vm/JIT.cpp

```diff
@@ -214,13 +214,20 @@ jllvm::JIT::~JIT()
 void jllvm::JIT::add(const ClassObject* classObject)
 {
     llvm::orc::IndirectStubsManager::StubInitsMap stubInits;
+    llvm::orc::SymbolMap methodGlobals;
     for (const Method& method : classObject->getMethods())
     {
         if (method.isAbstract())
         {
             continue;
         }
 
+        // Register the method in the JIT symbol table in case any code references it. This is done for methods as
+        // there is exactly one symbol per method but not for class objects, as there are infinitely many class objects
+        // due to being able to create array class objects of other class objects.
+        // Class objects are therefore created on demand in 'ClassObjectStubDefinitionsGenerator'.
+        methodGlobals[m_interner(mangleMethodGlobal(&method))] = llvm::JITEvaluatedSymbol::fromPointer(&method);
```

Comment:
Is there any difference to
```suggestion
        methodGlobals.insert({m_interner(mangleMethodGlobal(&method)), llvm::JITEvaluatedSymbol::fromPointer(&method)});
```?

Reply:
`llvm::orc::SymbolMap` is really a `llvm::DenseMap` which behaves like other C++ maps. In this specific case there is no difference but if the key-value pair existed previously, using `operator[]` would override the mapped value while `insert` would not do an insertion and keep the existing value. 

Do you have a peference for one of the two forms? I personally like the `operator[]` syntax more for this specific case

Reply:
I personally like insert better, since it is the only entry in the map.

Reply:
Alright